internal/rest/restgateway.go

```diff
@@ -78,7 +78,7 @@ func NewRESTGateway(config *conf.RESTGatewayConf) *RESTGateway {
 }
 
 func (g *RESTGateway) Init() error {
-	rpcClient, err := fabric.RPCConnect(g.config.RPC, g.config.MaxTXWaitTime)
+	rpcClient, identityClient, err := fabric.RPCConnect(g.config.RPC, g.config.MaxTXWaitTime)
```

Comment:
Is the REST Gateway the right place to initialize and own the pointer to this?

Reply:
fair question, the identity client may only get used in a small portion of the connector life cycle, however initializing the identity client requires a large amount of config information loading, and repeated config provider and context objects constructions, which makes the same process to construct the rpc client the ideal place to construct it.

conceptually, the rpc client (which sends grpc requests to a Fabric node) is peer to the identity client (which sends http requests to a Fabric CA node). so architecturally the two sitting next to each other is defensible.

Reply:
Yeah it makes sense that the RPC client and the Identity client are peers so they should sit next to each other. I'm just wondering if the RESTGateway is really the "parent" of those things. Maybe it makes sense for now, but possibly something to revisit later as fabconnect grows.